styles.css

```diff
@@ -0,0 +1,509 @@
+    font-size: 2rem;
+    color: #5722CD;
+    padding-top: 150px;
+}
+
+.main-section-one-inner-container h2 {
+    padding: 20px 0 20px 0;
+    font-weight: 500;
+}
+
+.main-section-one-inner-container p {
+    padding: 100px 0 100px 0;
+    color: #0C0C0C;
+}
+
+.main-section-one-inner-container p a {
+    text-decoration: none;
+    color: #5722CD;
+}
+
+.driehoek-twee{
+    width: 100vw;
+    height: 30px;
+    position: absolute;
+    bottom: 0;
+    border-right: 100vw solid transparent;
+    border-bottom: 30px solid #F3F6F9;
+}
+
+.main-section-one-outer-container::before {
+    content: 'hello';
+    position: absolute;
+    top: 0;
+    font-size: 25rem;
+    z-index: -1;
+    color: #F3F6F9;
+}
+
+/* --------------- 4.3 main projects ------------------- */
+
+.section-two-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background-color: #F3F6F9;
+    padding: 250px 50px 250px 50px;
+    flex-basis: 100vw;
+    position: relative;
+    z-index: 0;
+    align-items: center;
+}
+
+.main-section-two-outer-container {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 95vw;
+}
+
+.main-article-section-two {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.main-article-section-two-reverse {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    padding: 150px 0 150px 0;
+}
+
+.article-inner-container-img {
+    display: flex;
+    flex-basis: 50%;
+    flex-grow: 0;
+    flex-shrink: 1;
+    box-shadow: 0 35px 10px rgba(0.0, 0, 0, 0.03), 0 0 0 rgba(0, 0, 0, 0.00);
+}
+
+.work-image {
+    width: 100%;
+    flex-shrink: 1;
+}
+
+.article-inner-container-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 100px 0 100px;
+    flex-basis: 600px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    margin-left: 50px;
+    position: relative;
+}
+
+.article-inner-container-text::before {
+    content: '';
+    position: absolute;
+    display: block;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    background-image: url("assets/squiggle_line.svg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    background-position: bottom left;
+}
+
+.reverse {
+    margin: 0 50px 0 0;
+}
+
+.article-inner-container-text h2{
+    padding: 20px 0 80px 0;
+    font-size: 3rem;
+    position: relative;
+}
+
+.article-inner-container-text h2::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 80px;
+    height: 80px;
+    background-image: url("assets/squiggle_line.svg");
+    background-size: 100% 100%;
+}
+
+.article-inner-container-text p {
+    font-size: 1.5rem;
+    padding: 45px 0 10px 0;
+}
+
+
+.article-inner-container-text p a {
+    text-decoration: none;
+    color: #5722CD;
+    position: relative;
+}
+
+
+.article-inner-container-text p a::after {
+    content: "";
+    width: 15%;
```

Comment:
Wil je dat het streepje altijd 15% van de grootte van het woord aanneemt (dus groter bij `htmlcssweek4huiswerkconceptopdracht` en kleiner bij `klik hier`), of moet het streepje altijd even groot zijn?